docs/_getting-started/index.md

```diff
@@ -44,25 +44,31 @@ You can connect a Geth node to several different networks using the network name
 These include the main Ethereum network, [a private network](getting-started/private-net) you create,
 and three test networks that use different consensus algorithms:
 
--   **Ropsten**: Proof-of-work test network
--   **Rinkeby**: Proof-of-authority test network
--   **Görli**: Proof-of-authority test network
+-   **Ropsten**: Proof-of-work test network
+-   **Rinkeby**: Proof-of-authority test network
+-   **Görli**: Proof-of-authority test network
 
 For this guide, we use the Görli network.
 
+Because of the peer to peer nature of Ethereum the system running Geth has to:
+
+1. Have a publicly accessible IP address (it does not have the be the same as the private IP)
+2. Be open to access on port 30303 (TCP and UDP)
+
+
 ### Sync modes
 
 You can start Geth in one of three different sync modes using the `--syncmode "<mode>"`
 argument that determines what sort of node it is in the network.
 
 These are:
 
--   **Full**: Downloads all blocks (including headers, transactions, and receipts) and
-    generates the state of the blockchain incrementally by executing every block.
--   **Fast** (Default): Downloads all blocks (including headers, transactions and
-    receipts), verifies all headers, and downloads the state and verifies it against the
-    headers.
--   **Light**: Downloads all block headers, block data, and verifies some randomly.
+-   **Full**: Downloads all blocks (including headers, transactions, and receipts) and
+    generates the state of the blockchain incrementally by executing every block.
+-   **Fast** (Default): Downloads all blocks (including headers, transactions and
```

Comment:
Also: 
Snap: downloads all blocks (including headers, transactions and receipts), verifies all headers, and downloads state and verifies it against headers. The end result, and security model, is same as fast-sync, but the snap mode of download is a lot more efficent.

Reply:
Is there any reason to still use fast? Is snap the new default?